helix-core/src/comment.rs

```diff
@@ -345,17 +345,17 @@ mod test {
 
             let text = doc.slice(..);
 
-            let res = find_line_comment("//", text, 0..3);
+            let res = find_line_comment("#", text, 0..3);
```

Comment:
Hm... I don't really see the need of these kind of changes. It should be enough to just change the constant as you did.
May I ask why you did that?

Reply:
hmm, i ignored to change the token in my first commit and failed to pass [the workflow test suite](https://github.com/helix-editor/helix/actions/runs/11877333431/job/33096665578), so i replaced all the token from "//" to "#" in the tests

Reply:
> Hm... I don't really see the need of these kind of changes. It should be enough to just change the constant as you did. May I ask why you did that?

hmm, i ignored to change the token in my first commit and failed to pass [the workflow test suite](https://github.com/helix-editor/helix/actions/runs/11877333431/job/33096665578), so i replaced all the token from "//" to "#" in the tests

Reply:
That's really weird. If I run `cargo integration-test` on my machine with the following test function:
```rust
        #[test]
        fn not_commented() {
            // four lines, two space indented, except for line 1 which is blank.
            let doc = Rope::from("  1\n\n  2\n  3");

            let text = doc.slice(..);

            let res = find_line_comment("//", text, 0..3);
            // (commented = false, to_change = [line 0, line 2], min = col 2, margin = 0)
            assert_eq!(res, (false, vec![0, 2], 2, 0));
        }
```
everything works fine, as expected.

Reply:
okay there's the new comment 02162e998d9cf7815963786f790c0e0c22f3ea24, just keep it as it was

Reply:
> that's not weird

well, not for all tests but for this one, it is. There are no comment tokens, so the result should be the same that there isn't a comment token.

> other tests failed due to "//" is not replaced

aye

Reply:
> there's no reason to keep "//" here

hm... maybe for testing the `find_line_comment` for different comment tokens as well?

Anyways, you already applied the changes :)